charts/govuk-jobs/templates/content-store-mongo-to-postgres-cronjob.yaml

```diff
@@ -174,11 +174,10 @@ spec:
                   mountPath: /pg-dump
             - name: dump-postgresql
               image: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/github-cli:latest
-              command: ["pg_dump"]
+              command: ["bash"]
```

Comment:
Only other thing I wanted to check was that `bash` is actually the shell in the image - `sh` might have been more portable if not. But [it is bash](https://github.com/alphagov/govuk-infrastructure/blob/80331f8455453414c311740376bfb1c330a079dd/images/github-cli/Dockerfile#L1) so all good.